Week1/countryData.js

```diff
@@ -0,0 +1,37 @@
+const checkAndCreate = 'CREATE TABLE IF NOT EXISTS';
+const insert = `INSERT INTO country 
+                (Code, Name, Continent, Region, SurfaceArea, 
+                IndepYear, Population, LifeExpectancy, GNP, GNPOld, 
+                LocalName, GovernmentForm, HeadOfState, Capital) VALUES`;
+const CREATE_COUNTRY_TABLE = `
+                ${checkAndCreate} country(
+                Code  VARCHAR(3) NOT NULL DEFAULT '',
+                Name  VARCHAR(52) NOT NULL DEFAULT '',
+                Continent  enum('Asia','Europe','North America','Africa','Oceania','Antarctica','South America') NOT NULL,
+                Region  VARCHAR(26) NOT NULL DEFAULT '',
+                SurfaceArea  FLOAT(10,2) NOT NULL DEFAULT 0.00,
+                IndepYear  SMALLINT(6) DEFAULT NULL,
+                Population  INT(11) NOT NULL DEFAULT 0,
+                LifeExpectancy  FLOAT(3,1) DEFAULT NULL,
+                GNP  FLOAT(10,2) DEFAULT NULL,
+                GNPOld  FLOAT(10,2) DEFAULT NULL,
+                LocalName  VARCHAR(45) NOT NULL DEFAULT '',
+                GovernmentForm  VARCHAR(45) NOT NULL DEFAULT '',
+                HeadOfState  VARCHAR(60),
+                Capital  VARCHAR(20) NOT NULL,
+                Code2  VARCHAR(2) NOT NULL DEFAULT '',
+                PRIMARY KEY (Code)
+              )`;
+
+const INSERT_COUNTRY_QUERIES = `
```

Comment:
This is really nice, you have managed to use only one call to the server to insert every lines!